### DISCUSSION TITLE
Add Base network logo

### DISCUSSION THREAD

**Discussion by NandyBa** [family/connectkit#417] [Ideas]
Hi, it's possible to add Base chain and more specially Base network logo on Connectkit ?
![image](https://github.com/user-attachments/assets/a43f3e98-1fbe-4bc8-ae38-a239fc971da7)

We are required to use this chain for [this Merit campaign](https://governance.aave.com/t/arfc-merit-base-incentives-and-superfest-matching/18450) 

Best,
ACI team

**Comment by lochie** [comment]
Great suggestion @NandyBa! Have added in PR #418, will be available in the next patch which will be some time this week 👍

**Reply by NandyBa** [reply]
Thanks

**Reply by NandyBa** [reply]
Hi, gently reminder @lochie. This is not critical but that the last missing change needed for a internal PR for Merit front-end
